### PR TITLE
Update Share to WordPress dialog styling

### LINKED ISSUE #5126: Improve design styling of share-to-post dialog
### Steps to reproduce the behavior
1) From another app select a picture or text and click the Share action
2) Select WordPress

The site picker dialog functions well, but could probably use a design refresh.

image

### PR DESCRIPTION
**Pull request by tonyr59h**
Fixes #5126.

[Before and after screenshots.](https://cloudup.com/cEL8cz0MnDm)

To test:
* Go to Gallery or similar app
* Share a picture with WordPress

### CODE REVIEW

**Comment by nbradbury** [comment]
When trying to share a photo, hitting the Share button without selecting one of the radio buttons results in the toast shown below. This same toast always appears after attempting to share text (where the radio buttons are hidden).

screenshot_1495105934

Maybe

File: WordPress/src/main/java/org/wordpress/android/ui/ShareIntentReceiverActivity.java [hunk 1]
```diff
@@ -76,48 +75,77 @@ protected void onCreate(Bundle savedInstanceState) {
...
+     */
+    public void onCancelClicked(View view) {
+        finish();
     }
 
     private void finishIfNoVisibleBlogs() {
-        // If not logged in, then ask to log in, else inform the user to set at least one blog
-        // visible
+        // If not logged in, then ask to log in, else inform the user to set at least one blog visible
         if (FluxCUtils.isSignedInWPComOrHasWPOrgSite(mAccountStore, mSiteStore)) {
```
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/ShareIntentReceiverActivity.java]: This isn't a result of this PR, but this routine looks wrong to me. Shouldn't this be if **not** isSignedInWPComOrHasWPOrgSite?

**Comment by tonyr59h** [comment]
> Maybe we should simply default to having the "New post" radio button selected?

That sounds like a reasonable approach.

File: WordPress/src/main/java/org/wordpress/android/ui/ShareIntentReceiverActivity.java (hunk 1 above)
Comment (tonyr59h) [inline WordPress/src/main/java/org/wordpress/android/ui/ShareIntentReceiverActivity.java]: Yeah, looks like the `!` was [accidentally dropped](https://github.com/wordpress-mobile/WordPress-Android/blame/8ddb7cba951bb5329761945f06bbccf2109cfc60/WordPress/src/main/java/org/wordpress/android/ui/ShareIntentReceiverActivity.java#L111) during the FluxC integration process.

Comment (tonyr59h) [inline WordPress/src/main/java/org/wordpress/android/ui/ShareIntentReceiverActivity.java]: Fixed in eebb515831dd68687a5f058202e71190174ec8f5

**Comment by nbradbury** [comment]
Looks good! :shipit:

**Pull request merged**